server/channels/app/plugin_hooks_test.go

```diff
@@ -1704,10 +1701,7 @@ func TestHookPreferencesHaveChanged(t *testing.T) {
 		require.Nil(t, err)
 
 		// Hooks are run in a goroutine, so wait for those to complete
-		time.Sleep(1 * time.Second)
-
-		mockAPI.AssertCalled(t, "LogDebug", "category=test_category name=test_name_1 value=test_value_1")
-		mockAPI.AssertCalled(t, "LogDebug", "category=test_category name=test_name_2 value=test_value_2")
```

Comment:
Why the removal?

Reply:
Because `mockAPI.AssertExpectations(t)` is anyways called on L1696. This is redundant.